app-misc/terminal-colors/terminal-colors-3.0.2.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="ncurses"
+
+inherit python-r1
+
+DESCRIPTION="A tool to display color charts for 8, 16, 88, and 256 color terminals"
+HOMEPAGE="http://zhar.net/projects/shell/terminal-colors"
+SRC_URI="https://github.com/eikenb/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}"
+
+src_install() {
+	mv "${S}/${P}/${PN}" "${S}" || die "moving the script failed"
+	rm -rf "${S}/${P}" || die "rm failed"
```

Comment:
I'm sorry but.. what is going on in here? Can you explain what you're doing and why.

Reply:
The line 23, moves the script from the opened folder to the work directory, and in line 24, it deletes the opened folder, which contains the README, license and other stuff that are not necessary for the package.

Reply:
Okay, I think I see what's happening here. The previous distfile just provided the final terminal-colors file, while the real upstream one provides much more, including a setup.py. You're already calling `python_foreach_impl python_newscript ${PN} ${PN}` to just install what you want. But for that to work, I guess we'll have to...